worker/src/RTC/TransportCongestionControlServer.cpp

```diff
@@ -264,7 +285,7 @@ namespace RTC
 		}
 	}
 
-	inline void TransportCongestionControlServer::SendTransportCcFeedback()
```

Comment:
Unrelated by let's remove step by step those ugly `inline` usages. We are in 2024.